README.rst

```diff
@@ -152,6 +152,11 @@ All Python versions should support the following in-line code:
   python -c 'import urllib; print urllib.urlopen("https://bootstrap.saltstack.com").read()' > bootstrap-salt.sh
   sudo sh bootstrap-salt.sh git develop
 
+or with python version 3:
```

Comment:
I would change the line up above that says "All Python Version should support" and make it more obvious/organized that one is python2 and one is python3.  I'd add the sudo line so they look more similar as well.